src/main/java/user/UserProcessor.java

```diff
@@ -0,0 +1,62 @@
+package user;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Objects;
+
+import db.DataBase;
+import model.User;
+
+public class UserProcessor {
```

Comment:
UserProcessor는 유틸 클래스인가요?
```suggestion
public final class UserProcessor {
	private UserProcessor() { }
```

그렇다면 상속과 인스턴스 생성을 막아줄 필요가 있어 보이네요